suites/teuthology/rgw/tasks/s3tests-fastcgi.yaml

```diff
@@ -0,0 +1,24 @@
+# this runs s3tests against rgw, using mod_fastcgi
+roles:
+- [mon.a, mon.c, osd.0, osd.1, osd.2, client.0]
+- [mon.b, osd.3, osd.4, osd.5, client.1]
+
+tasks:
+- install:
+    branch: firefly
+- ceph:
+- rgw: [client.0]
+- s3tests:
+    client.0:
+      rgw_server: client.0
+      force-branch: wip-10551-firefly-original
```

Comment:
Just a reminder that we may want to remove this before merging as the branch might go away.